### DISCUSSION TITLE
Can't add school email to GitHub Student Developer Pack application

### DISCUSSION THREAD

**Discussion by jhould007** [community/community#19741] [GitHub Education] [answered]
I am trying to apply for a student developer pack, and when I submitted with my personal email, it said a school email was required. I tried putting my school email in, but since I have a separate school account that my university required me to create that is connected to my school email, I am unable to add my school email to my personal account, which is the account I want to use GitHub Copilot on. Therefore, I am completely unable to submit an application for the SDP. Does anyone have any suggestions? 

I don't really use my school account anymore, but I would prefer to keep it if possible because there's a lot of projects on there that I would like to preserve for posterity. 

EDIT: I was able to solve the problem by adding another alternate email to my school account, removing my school email from it, and then adding my school email to my personal account. I suggest that those who are having the same issue do this. It's pretty easy to create a secondary email account through Yahoo, for example.

**Comment by D7EAD** [comment, marked as answer]
Hi!

I'd suggest you launch a ticket over to Github Education's Support Engineers regarding this matter. You can do so [here](https://support.github.com/contact/education).

**Comment by Fabio53443** [comment]
Hi there! 
Due to school security policies on official accounts, i wasn't able to add my school email either but i was still able to get in the sdp by providing a screenshot of the school web portal and one of a school letterheaded document with my name. If your University gives you an official ID, you should probably use that!
Hope this helped

**Reply by PerezChris99** [reply]
I believe you can reach out to the Github Education support engineers, they'll help out

**Reply by KhoubziAbdelhak** [reply]
can you send me the email to contact them please

**Comment by JhonnyLusonode** [comment]
Hello @jhould007  there is already a Discussion about in #19050 
Checking it out maybe there are all the answers you need.

If you find this answer useful please consider marking it as "Answered".
(btw, this would help me to get the badge. Thank you! 😁)

**Comment by PerezChris99** [comment]
Kindly reach out to the Github education support engineering team
